hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -261,6 +261,7 @@ private ContainerCommandResponseProto dispatchRequest(
         // If container does not exist, create one for WriteChunk and
         // PutSmallFile request
         responseProto = createContainer(msg);
+        metrics.incContainerOpsMetrics(Type.CreateContainer);
```

Comment:
I guess it should also call `incContainerOpsLatencies` at some point.

Reply:
Oh! sorry i should use _incContainerOpsLatencies_ instead, let me update it!

Reply:
update: here we shall use both incContainerOpsMetrics + incContainerOpsLatencies.